src/js/form-builder.js

```diff
@@ -98,7 +96,7 @@ const FormBuilder = function(opts, element) {
     }
     let icon = custom.icon || controlClass.icon(type);
     let label = custom.label || controlClass.label(type);
-    let iconClassName = !icon ? custom.iconClassName || `icon-${type}` : '';
+    let iconClassName = !icon ? custom.iconClassName || `icon-${type.replace(/-[\d]{4}$/, '')}` : '';
```

Comment:
@kevinchappell is this `icon-${type.replace(/-[\d]{4}$/, '')}` there to handle the custom fields?

If so a better way to write this section may be:

```
let iconClassName = '';
if (!icon) {
   icon = custom ? custom.iconClassName || custom.type : `icon-${type}`;
}